airflow/providers/google/cloud/triggers/bigquery.py

```diff
@@ -205,7 +221,12 @@ class BigQueryGetDataTrigger(BigQueryInsertJobTrigger):
         (default: False).
     """
 
-    def __init__(self, as_dict: bool = False, selected_fields: str | None = None, **kwargs):
+    def __init__(
+        self,
+        as_dict: bool = False,
+        selected_fields: str | None = None,
+        **kwargs,
```

Comment:
```suggestion
        **kwargs
```

Reply:
I'm think we're good with keeping it as what it used to be

Reply:
Changed it.